src/components/chatbox/welcome-message.tsx

```diff
@@ -26,80 +46,43 @@ export const WelcomeMessage = () => {
 						weight="medium"
 						size={400}
 					>
-						Welcome to Developer AI
+						Developer AI
 					</Heading>
 					<Text /* Body/100/Medium */
 						className={s.faint}
 						size={100}
 						weight="medium"
 					>
-						Your personal AI-powered assistant, we’re ready to help you get the
-						most out of Developer. Let’s get started on this journey together.
+						Your AI-powered companion for finding reference materials,
+						architectural guidance, and product examples from HashiCorp. Ask me
+						anything to get started.
 					</Text>
 				</div>
 				<div className={s.spacer} />
 			</div>
 			<div className={classNames(s.col, s.right)}>
-				<div className={s.row}>
-					<IconUser16 />
-					<div>
-						<Heading /* Display/100/Medium */
-							className={s.strong}
-							level={4}
-							weight="medium"
-							size={100}
-						>
-							Personalized recommendations
-						</Heading>
-						<Text /* Body/100/Regular */
-							size={100}
-							weight="regular"
-							className={s.faint}
-						>
-							Coming soon...
-						</Text>
-					</div>
-				</div>
-				<div className={s.row}>
-					<IconDiscussionCircle16 />
-					<div>
-						<Heading /* Display/100/Medium */
-							className={s.strong}
-							level={4}
-							weight="medium"
-							size={100}
-						>
-							Natural language conversations
-						</Heading>
-						<Text /* Body/100/Regular */
-							className={s.faint}
-							size={100}
-							weight="regular"
-						>
-							Coming soon...
-						</Text>
+				{rightColumn.map(({ icon, heading, text }, i) => (
```

Comment:
DRY'ing up some markup behind `rightColumn.map`

Also make it easy to view copy that needs to be updated before production rollout